doc/contributing.rst

```diff
@@ -553,20 +553,40 @@ To keep the tool user friendly, try to avoid making changes that are not
 backward compatible, i.e. changes that require users to change their existing
 recipes, diagnostics, configuration files, or scripts.
 
-If you really must change the public interfaces of the tool, always discuss this
-with the `@ESMValGroup/esmvaltool-coreteam`_.
-Try to deprecate the feature first by issuing a :py:class:`DeprecationWarning`
-using the :py:mod:`warnings` module and schedule it for removal three
-`minor versions <https://semver.org/>`__ from the latest released version.
-For example, when you deprecate a feature in a pull request that will be
-included in version 2.3, that feature could be removed in version 2.5.
+If you really must change the public interfaces of the tool, always discuss
+this with the `@ESMValGroup/esmvaltool-coreteam`_. Try to deprecate the
```

Comment:
Please [always start a new sentence on a new line in rst files](https://docs.esmvaltool.org/en/latest/community/code_documentation.html#documentation). This results in much more readable diffs.